metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/force_delete_test.clj

```diff
@@ -124,3 +128,52 @@
                              "does not exist.")
                         svc-concept-id non-extant-revision)]
                (:errors response)))))))
+
+(deftest force-delete-variable-with-associations
+  (let [coll (util/create-and-save-collection "REG_PROV" 1)
+        coll-concept-id (:concept-id coll)
+        var-concept (util/create-and-save-variable "REG_PROV" 1 3)
+        var-concept-id (:concept-id var-concept)
+        var-assn (util/create-and-save-variable-association
+                  coll var-concept 1)
+        var-assn-concept-id (:concept-id var-assn)]
+    (testing "initial conditions"
+      ;; creation results as expected
+      (is (= 3 (:revision-id var-concept)))
+      ;; variable revisions in db
+      (is (= 3
+             (count (:concepts (util/find-concepts :variable)))))
+      (is (= 200 (:status (util/get-concept-by-id-and-revision
+                           var-concept-id 3))))
+      (is (= 200 (:status (util/get-concept-by-id-and-revision
```

Comment:
For completeness, add checking of revision 1 exists in all the testing cases.

Reply:
Done.